app/utils/DirectoryCloneUtils.js

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs';
+import path from 'path';
+
+/**
+ * Clones the directory structure from a source directory to a target directory
+ * without copying any files. Hidden directories (starting with '.') are ignored.
+ * 
+ * @param {string} sourceDir - The source directory to clone from
+ * @param {string} targetDir - The target directory to clone to
+ * @returns {Promise<void>}
+ */
+export const cloneDirectoryStructure = async (sourceDir, targetDir) => {
+  try {
+    // Create the target directory if it doesn't exist
+    if (!fs.existsSync(targetDir)) {
+      fs.mkdirSync(targetDir, { recursive: true });
+    }
+
+    // Read all items in the source directory
+    const items = fs.readdirSync(sourceDir);
+
+    // Process each item
+    for (const item of items) {
+      // Skip hidden directories (starting with '.')
+      if (item.startsWith('.')) {
+        continue;
+      }
+
+      const sourcePath = path.join(sourceDir, item);
+      const targetPath = path.join(targetDir, item);
+
+      // Check if the item is a directory
+      const stats = fs.statSync(sourcePath);
+      if (stats.isDirectory()) {
+        // Create the directory in the target
+        fs.mkdirSync(targetPath, { recursive: true });
+        
+        // Recursively clone subdirectories
+        await cloneDirectoryStructure(sourcePath, targetPath);
+      }
+      // We intentionally skip files as per requirements
+    }
+  } catch (error) {
+    throw new Error(`Failed to clone directory structure: ${error.message}`);
+  }
+};
+
+/**
+ * Creates a new StatWrap configuration folder in the target directory
+ * 
+ * @param {string} targetDir - The target directory where the .statwrap folder will be created
+ * @returns {Promise<void>}
+ */
+export const createStatWrapConfig = async (targetDir) => {
+  try {
+    const configDir = path.join(targetDir, '.statwrap');
+    fs.mkdirSync(configDir, { recursive: true });
+    
+    // Create basic configuration files
+    // This is just a placeholder - you'll need to implement the actual config creation
+    // based on how StatWrap creates config files for new projects
+    const configFile = path.join(configDir, 'config.json');
```

Comment:
I'm wondering why we would create this file instead of the actual .statwrap-project.json.  This will leave an extra file around that is not usable.  Would it be possible to create a bare minimum .statwrap-project.json file instead?